src/_pytest/pytester.py

```diff
@@ -81,7 +81,9 @@ def get_open_files(self):
 
     def _exec_lsof(self):
         pid = os.getpid()
-        return subprocess.check_output(("lsof", "-Ffn0", "-p", str(pid))).decode()
+        return subprocess.check_output(
+            ("lsof", "-Ffn0", "-p", str(pid)), stderr=os.devnull
```

Comment:
I think this needs to be

```
with open(os.devnull, 'wb') as devnull:
  ... stderr=devnull
```

Sorry I think I might've broken this (and if this is wrong I'm typing it on my phone :laughing:)

Reply:
`subprocess.DEVNULL` is python 3 only, your example looks roughly correct and certainly gives the right idea
we shouldu se it and add the subprocess.DEVNULL comment to clean up in future